fontforge/featurefile.c

```diff
@@ -1,6 +1,7 @@
 /* Copyright (C) 2000-2012 by George Williams */
 /* Copyright (C) 2012-2013 by Khaled Hosny */
 /* Copyright (C) 2013 by Matthew Skala */
+/* Copyright (C) 2020 by Rajeesh KV */
```

Comment:
I would prefer you instead add your name to the AUTHORS file, if that's OK with you. Same for `fontview.c`.

Reply:
Certainly. I wasn't sure.
But now that I checked `AUTHORS` file, I’m already there `Rajeesh K Nambiar` for an ancient  contribution.